skimage/future/_thin_plate_splines.py

```diff
@@ -0,0 +1,323 @@
+import numpy as np
+import scipy as sp
+
+from .._shared.utils import check_nD
+
+
+class TpsTransform:
```

Comment:
Should be `TPSTransform`, since `TPS` is an acronym.

But why not `ThinPlateSplineTransform`?

Reply:
`ThinPlateSplineTransform` 👍 is more explicit
@mkcor @lagru ?

Reply:
Happy to expand the acronym. :)

Reply:
:+1: